server/chat-commands.js

```diff
@@ -3740,6 +3740,39 @@ const commands = {
 	},
 	importinputloghelp: [`/importinputlog [inputlog] - Starts a battle with a given inputlog. Requires: + % @ & ~`],
 
+	acceptdraw: 'acceptdraw',
+	offerdraw(target, room, user, connection, cmd) {
+		const battle = room.battle;
+		const playerIDS = Object.keys(battle.playerTable);
```

Comment:
despite renaming `toID`, I think `playerIds` would still be more typical for the codebase.

Reply:
I like camelCase so went with `playerIds` :p

Reply:
I'd imagine `playerIDs` works fine.